### PR TITLE
Performance audit

### PR DESCRIPTION
**Pull request by alphayowakarindi**
At this juncture of development I implemented the following:
- Added email column in owners table
- Increased data in owners and visits table.
- Created indexes to reduce execution time and here are the results: [screenshot 1](https://github.com/alphayowakarindi/vet-clinic-database/blob/performance-audit/executionReduction/after-decrasing-speed.png) and [screenshot 2](https://github.com/alphayowakarindi/vet-clinic-database/blob/performance-audit/executionReduction/before-speed-decrease.png)

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -63,4 +63,19 @@ CREATE TABLE visits(
...
+-- Performance audit
+-- Add an email column to your owners table
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+-- This will add 3.594.280 visits considering you have 10 animals, 4 vets, and it will use around ~87.000 timestamps (~4min approx.)
+INSERT INTO visits (animal_id, vet_id, date_of_visit) SELECT * FROM (SELECT id FROM animals) animal_ids, (SELECT id FROM vets) vets_ids, generate_series('1980-01-01'::timestamp, '2021-01-01', '4 hours') visit_timestamp;
+
+-- This will add 2.500.000 owners with full_name = 'Owner <X>' and email = 'owner_<X>@email.com' (~2min approx.)
+insert into owners (full_name, email) select 'Owner ' || generate_series(1,2500000), 'owner_' || generate_series(1,2500000) || '@mail.com';
+
```
Comment (m05e5) [inline schema.sql]: - [x] Great job. As mentioned in the requirements, make sure to have your INSERT INTO in the data.sql file
image

**Review by m05e5 — CHANGES_REQUESTED** [review]
Hi @alphayowakarindi , 

Good job so far. I really like what you did!
There are some issues that you still need to work on to go to the next project but you are almost there!
mario_ladders


## Highlights


## Required Changes ♻️
* You really did a great job but there are some features you need to improve

- [ ] Please check the comments below

_I'm sure you can do it just keep it up_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_@m05e5_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by alphayowakarindi** [comment]
> Hi @alphayowakarindi. you moved the 'visit' request but not the 'owners'

Hi @m05e5, thank you for the feedback to make my code better. If I got you correct, do you mean I did not move this line `insert into owners (full_name, email) select 'Owner ' || generate_series(1,2500000), 'owner_' || generate_series(1,2500000) || '@mail.com';` to `data.sql`

**Review by m05e5 — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @alphayowakarindi and @basitali111    🔥 

Good job on implementing the requirements. Well Done.
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

goodjob


Now you can merge this branch 🚀 

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

_m05e5_

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**